app/queries/app/servers.ts

```diff
@@ -37,6 +37,12 @@ export const queryServerByIdentifier = async (appDatabase: Database, identifier:
     }
 };
 
+export const queryServerByDisplayName = async (appDatabase: Database, displayName: string) => {
+    const servers = await queryAllServers(appDatabase);
+    const server = servers.find((s) => s.displayName.toLowerCase() === displayName.toLowerCase());
```

Comment:
this is done this way instead of a direct query to the db using the `display_name` column to ensure the name is the same regardless of case sensitivity